lib_eio_luv/eio_luv.ml

```diff
@@ -455,6 +470,28 @@ module Low_level = struct
         t.fd <- `Closed;
         Eio.Switch.remove_hook t.release_hook;
         fd
+
+    let kind_map =
+      (* we only need 12, but 13 maps directly *)
+      let km = Array.init 13 (fun _ -> `Unknown) in
+      Array.set km 0o1  `Fifo;
+      Array.set km 0o2  `Character_special;
+      Array.set km 0o4  `Directory;
+      Array.set km 0o6  `Block_device;
+      Array.set km 0o10 `Regular_file;
+      Array.set km 0o12 `Symbolic_link;
+      Array.set km 0o14 `Socket;
+      km
+
+    let kind_of_mode (mode : int) : Eio.Fs.kind =
+      if Sys.os_type <> "Unix" then
+        `Unknown
```

Comment:
What happens on Windows?

Reply:
No clue :man_shrugging:, so I chose to be conservative until we could test it on windows.